src/NuGet.Services.EndToEnd/project.json

```diff
@@ -1,17 +1,16 @@
 ﻿{
   "dependencies": {
     "Microsoft.AspNetCore.WebUtilities": "1.1.1",
-    "Microsoft.Extensions.Configuration.Binder": "1.0.0",
-    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
-    "Microsoft.Extensions.Configuration.Json": "1.0.0",
-    "Microsoft.Extensions.Primitives": "1.1.0",
-    "Newtonsoft.Json": "9.0.1",
+    "Microsoft.Extensions.Configuration.Binder": "1.1.2",
+    "Microsoft.Extensions.Configuration.FileExtensions": "1.1.2",
+    "Microsoft.Extensions.Configuration.Json": "1.1.2",
+    "Microsoft.Extensions.Primitives": "1.1.1",
+    "Newtonsoft.Json": "10.0.2",
     "NuGet.Frameworks": "4.9.2",
     "NuGet.Packaging": "4.9.2",
     "NuGet.Protocol": "4.9.2",
-    "NuGet.Services.AzureManagement": "2.2.5",
-    "NuGet.Services.Configuration": "2.2.4",
-    "NuGet.Services.KeyVault": "2.2.4",
+    "NuGet.Services.AzureManagement": "2.51.0-master-2758102",
```

Comment:
Please pull in a stable version

Reply:
Generate and use the stable version 2.51.0.
Test against the E2E pipeline successfully:
NuGet.Services.EndToEnd - Run end-to-end tests: 57391/57392.